Tools/TestWebKitAPI/Tests/WGSL/ConstLiteralTests.cpp

```diff
@@ -68,7 +69,7 @@ TEST(WGSLConstLiteralTests, BoolLiteral)
         const auto& intLiteral = downcast<AST::BoolLiteral>(expr);
         EXPECT_EQ(intLiteral.value(), testCase.expectedValue);
         auto inputLength = testCase.input.length();
-        EXPECT_EQ(intLiteral.span(), SourceSpan(1, inputLength, inputLength, 0));
+        EXPECT_EQ(intLiteral.span(), SourceSpan(1, 0, 0, inputLength));
```

Comment:
I spent a fair amount of time trying to make the tests pass without updating the expectations, but when I though about it, I think the new behavior is correct: shouldn't the span of a token be where it starts, not where it ends? I think this change is reasonable, but we can always change it later to match the previous behavior. Anything seems better than the current behavior of everything pointing to the end of the file.